packetbeat/protos/mongodb/mongodb.go

```diff
@@ -251,74 +251,65 @@ func (mongodb *Mongodb) onResponse(conn *mongodbConnectionData, msg *mongodbMess
 	mongodb.responses.Put(key, msg)
 }
 
-func (mongodb *Mongodb) onTransComplete(requ, resp *mongodbMessage) {
+func (mongodb *mongodbPlugin) onTransComplete(requ, resp *mongodbMessage) {
 	trans := newTransaction(requ, resp)
-	debugf("Mongodb transaction completed: %s", trans.Mongodb)
+	debugf("Mongodb transaction completed: %s", trans.mongodb)
 
 	mongodb.publishTransaction(trans)
 }
 
 func newTransaction(requ, resp *mongodbMessage) *transaction {
-	trans := &transaction{Type: "mongodb"}
+	trans := &transaction{}
 
 	// fill request
 	if requ != nil {
-		trans.tuple = requ.TCPTuple
-
-		trans.Mongodb = common.MapStr{}
+		trans.mongodb = common.MapStr{}
 		trans.event = requ.event
 		trans.method = requ.method
 
-		trans.cmdline = requ.CmdlineTuple
-		trans.ts = requ.Ts
-		trans.Ts = int64(trans.ts.UnixNano() / 1000) // transactions have microseconds resolution
```

Comment:
these fields have been set, but never been included in the event being generated (otherwise compilation would fail).